platform-api/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/UserService.java

```diff
@@ -424,6 +427,40 @@ public UserDescriptor getByName(@ApiParam(value = "User email")
         return ImmutableMap.of(USER_SELF_CREATION_ALLOWED, Boolean.toString(userSelfCreationAllowed));
     }
 
+    /**
+     * Get all users.
+     *
+     * @param maxItems
+     *         the maximum number of users to return
+     * @param skipCount
+     *         the number of users to skip
+     * @return list of users
+     * @throws BadRequestException
+     *         when {@code #maxItems} or {@code #skipCount} are incorrect
+     * @throws ServerException
+     *         when some error occurred while retrieving users
+     */
+    @GET
+    @Path("/findall")
+    @GenerateLink(rel = "get all users")
+    @RolesAllowed({"system/admin", "system/manager"})
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = "Get all users",
+                  notes = "Get all users in the system. Roles allowed: system/admin, system/manager.")
+    @ApiResponses({@ApiResponse(code = 200, message = "OK"),
+                   @ApiResponse(code = 400, message = "Bad Request"),
+                   @ApiResponse(code = 500, message = "Internal Server Error")})
+    public List<UserDescriptor> getAll(@ApiParam(value = "Max items") @QueryParam("maxItems") @DefaultValue("30") int maxItems,
```

Comment:
It should be another Service with path like
/user/admin
also we should avoid using verbs in REST methods (resources) addressing 
so
@GET
 +    @Path("/findall") - not good

In this case I would say we do not need a Path at all.